shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/SessionCardView.kt

```diff
@@ -1,40 +1,36 @@
 package org.jetbrains.kotlinconf
 
-import io.ktor.util.date.GMTDate
-import org.jetbrains.kotlinconf.utils.dayAndMonth
-import org.jetbrains.kotlinconf.utils.time
+import kotlinx.datetime.LocalDateTime
+import org.jetbrains.kotlinconf.utils.DateTimeFormatting
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
 
 data class SessionCardView(
     val id: SessionId,
     val title: String,
     val speakerLine: String,
     val locationLine: String,
-    val startsAt: GMTDate,
-    val endsAt: GMTDate,
+    val startsAt: LocalDateTime,
+    val endsAt: LocalDateTime,
     val state: SessionState,
     val speakerIds: List<SpeakerId>,
     val vote: Score?,
-    val timeLine: String = buildString {
-        append(startsAt.dayAndMonth())
-        append(", ")
-        append(startsAt.time())
-        append("-")
-        append(endsAt.time())
-    },
     val isFavorite: Boolean,
     val description: String,
     val tags: List<String>,
-    val badgeTimeLine: String = buildString {
-        append(startsAt.time())
-        append("-")
-        append(endsAt.time())
-    },
-    val isLightning: Boolean = endsAt.timestamp - startsAt.timestamp <= 15 * 60 * 1000,
     val startsInMinutes: Int?,
-)
+) {
+    val timeLine: String = DateTimeFormatting.dateAndTime(startsAt)
+    val badgeTimeLine: String = DateTimeFormatting.timeToTime(startsAt, endsAt)
+    val isLightning: Boolean = endsAt - startsAt <= LIGHTNING_TALK_LIMIT
+}
 
 val SessionCardView.isLive get() = state == SessionState.Live
 val SessionCardView.isUpcoming get() = state == SessionState.Upcoming
 val SessionCardView.isPast get() = state == SessionState.Past
 
-val Session.isLightning: Boolean get() = endsAt.timestamp - startsAt.timestamp <= 15 * 60 * 1000
+val Session.isLightning: Boolean
+    get() = endsAt - startsAt <= LIGHTNING_TALK_LIMIT
```

Comment:
minor: `isLightning` check is duplicated

Reply:
We need this both on `Session` and `SessionCardView`, I think with the limit variable extracted at least it's alright like this